examples/replication_configuration/main.tf

```diff
@@ -24,18 +24,18 @@ module "replication_role" {
 module "source_kms_key" {
   #checkov:skip=CKV_TF_1: "Ensure Terraform module sources use a commit hash"
   source           = "boldlink/kms/aws"
-  description      = "kms key for ${local.source_bucket}"
+  description      = "kms key for ${var.source_bucket}"
```

Comment:
This should be locked down to a version, if we introduce breaking changes, it breaks the example.

Also, the `CKV_TF_1` should be global, it should be on `.checkov.yaml` file

Reply:
Version now locked and exception moved to file

Reply:
Code still there